coverage/summary.py

```diff
@@ -30,6 +30,112 @@ def writeout(self, line):
         self.outfile.write(line.rstrip())
         self.outfile.write("\n")
 
+    def _report_text(self, header, lines_values, total_line, end_lines):
+        "internal method to print report data in text format"
+        # Prepare the formatting strings, header, and column sorting.
+        max_name = max([len(fr.relative_filename()) for (fr, analysis) in \
+            self.fr_analysis] + [5]) + 2
+        n = self.config.precision
```

Comment:
It seems odd to me to be looking at `config.precision` here, since you are getting all of the coverage totals as strings.  You shouldn't need to think about the precision, just deal with the strings as they are, sort of how `max_name` is calculated.